apps/cookbook/src/app/showcase/slides-showcase/slides-showcase.component.html

```diff
@@ -1,39 +1,84 @@
-<div class="example">
-  <p>
-    <em>
-      <strong>Please note:</strong>
-      The slides do not like beeing switched between chrome-devtools mobileview and standard
-      webview. Please refresh your browser after changing from web to mobile and visa versa
-    </em>
-  </p>
-  <cookbook-example-viewer [html]="slidesExample.template" [ts]="slidesExample.codeSnippet">
-    <cookbook-slides-example #slidesExample></cookbook-slides-example>
-  </cookbook-example-viewer>
-
-  <h4>Slides properties:</h4>
-  <cookbook-api-description-properties
-    [properties]="properties"
-  ></cookbook-api-description-properties>
-  <br />
-  <h4>Slides Events:</h4>
-  <cookbook-api-description-events [events]="events"></cookbook-api-description-events>
-  <p>
-    <em>
-      <strong>Note:</strong>
-      Loading lots of data in many cards can result in an unwanted pause while the slides content
-      are beeing rendered. (The data is recieved, spinner dissapears but seconds passes before the
-      cards are rendered) The notations:
-      <br />
-      <code>
-        *kirbySlide="let slide; let i = index"
-        <br />
-        [slideIndex]="i"
-        <br />
-        [activeSlideIndex]="selectedSlideIndex
-        <br />
-      </code>
-      gives you means to pass info to each slide´s content about what index it has and compare that
-      to the activeSlideIndex, and thereby a strategy to render only a few cards in advance.
-    </em>
-  </p>
-</div>
+<p>
+  Slides are a collection of cards you can navigate horizontally in, either swipe or click. Slides
+  are great to contain content without using a lot of space, but the content has to be closely
+  related.
+</p>
```

Comment:
Should be aligned with the [design specs](https://zpl.io/EmOPOge):
```suggestion
<p>
 Slides are a collection of cards you can navigate horizontally in, swipe on touch devices and click navigation on desktop.
</p>
<h2>Usage</h2>
<p>
  Slides are great to gather and contain content without using a lot of space, but the content has to be closely related.<br /> 
  Only use slides if you have 3 or more cards. If you have a lot of content, or if it takes more than four interactions (swipe or clicks), consider using a list format instead.<br /> 
Slides are best suited for inspirationel content where users are expected to be in a explorative mode.
</p>
```